static/app/components/group/tagFacets/tagFacetsDistributionMeter.tsx

```diff
@@ -263,8 +263,8 @@ const TagSummary = styled('div')`
   margin-bottom: ${space(2)};
 `;
 
-const TagHeader = styled('span')<{clickable?: boolean}>`
-  ${p => (p.clickable ? 'cursor: pointer' : null)};
+const TagHeader = styled('span')`
```

Comment:
This shouldn't be a `span` if it's clickable. There's no a11y with a bare span.

Where is this element used? i can't seem to find it